web/views.py

```
@@ -3101,3 +3105,44 @@ def get_object(self):
+        if form.is_valid():
+            certificate = form.save(commit=False)
+            certificate.user = request.user
+            certificate.save()
+            messages.success(request, "Certificate uploaded successfully!")
+            return redirect("certificate_list")
+    else:
+        form = CertificateForm()
+    
+    return render(request, "certificates/upload.html", {"form": form})
+
+from django.contrib.auth import get_user_model, login
```

Ruff (0.8.2)

Redefinition of unused `get_user_model` from line 15

Remove definition: `get_user_model`

(F811)